sourcegraph/sourcegraph
# Code Insights: Reuse series picker in compute insight creation UI page

Part of https://github.com/sourcegraph/sourcegraph/issues/37965
Fixes https://github.com/sourcegraph/sourcegraph/issues/38132 https://github.com/sourcegraph/sourcegraph/issues/37785

## Background 
<img width="1160" alt="Screenshot 2022-07-05 at 13 29 27" src="https://user-images.githubusercontent.com/18492575/177383569-76f20111-0604-4a01-afd9-ab0def66a4e3.png">

This PR adopts the 'FormSeries' component and reuses it on the compute-powered insight page.

## Test plan
- Make sure that all interactions with data series on the search-based creation UI page work correctly (edit data series, remove, add new one)
- Make sure that this logic works in the same way for the compute-powered insight creation UI



## App preview:

- [Web](https://sg-web-vk-re-use-series-picker-for.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-wboejamnyl.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
